### PR TITLE
Put our JES usage on a diet

### PR DESCRIPTION
**Pull request by geoffjentry**
Now that we know it was Centaur after all ...

### CODE REVIEW

File: src/bin/travis/resources/centaur.wdl [hunk 1]
```diff
@@ -22,7 +22,7 @@ task centaur {
 
     runtime {
         docker: "geoffjentry/centaur-cromwell:latest"
```
Comment (mcovarr) [inline src/bin/travis/resources/centaur.wdl]: Is there a Tech Debt ⚓ ticket to have this image somewhere not under your personal Docker account?

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/cromwell/pull-request/1295/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/cromwell)

File: src/bin/travis/resources/centaur.wdl (hunk 1 above)
Comment (geoffjentry) [inline src/bin/travis/resources/centaur.wdl]: No. When we first did it I asked if people cared and they said no.

What's way worse than this is that the encrypted bundle for travis, the original files live & knowledge of how to build it are only on my laptop

Comment (mcovarr) [inline src/bin/travis/resources/centaur.wdl]: Blurgh.  Would you mind writing a ticket for that?  I'd do it myself but I don't know all the things to ask to be done.  😄

Comment (geoffjentry) [inline src/bin/travis/resources/centaur.wdl]: Oh, I mind!

**Pull request merged**